js/client-chat.js

```diff
@@ -652,9 +652,14 @@
 				if (target.indexOf(',') > -1) {
 					var targets = target.match(/([^,]+?({\d*,\d*})?)+/g);
 					// trim the targets to be safe
+					var validTargets = [];
 					for (var i = 0, len = targets.length; i < len; i++) {
 						targets[i] = targets[i].replace(/\n/g, '').trim();
+						if (targets[i]) {
+							validTargets.push(targets[i]);
+						}
 					}
+					targets = validTargets;
 					switch (targets[0]) {
 					case 'add':
 						for (var i = 1, len = targets.length; i < len; i++) {
```

Comment:
Why not just do `if (!targets[i]) continue` in this loop?

Wouldn't require an extra allocation and would also use a lot less code.